readthedocs/templates/support.html

```diff
@@ -1,76 +0,0 @@
-{% extends "base.html" %}
-
-{% load i18n %}
-{% load core_tags %}
-
-
-{% block title %}Support{% endblock %}
-
-{% block content %}
-<div id="support_page">
-{% filter restructuredtext %}
```

Comment:
I think this was the only place were this filter was used. Maybe it's time to kill it?